src/families/elrond/api/sdk.js

```diff
@@ -0,0 +1,142 @@
+// @flow
+
+import { BigNumber } from "bignumber.js";
+import ElrondApi from "./apiCalls";
+import type { Transaction } from "../types";
+import type { Operation, OperationType } from "../../../types";
+import { getEnv } from "../../../env";
+import { encodeOperationId } from "../../../operation";
+
+const ELROND_API_ENDPOINT = () => getEnv("ELROND_API_ENDPOINT");
+
+let api = new ElrondApi(ELROND_API_ENDPOINT());
+
+/**
+ * Get account balances and nonce
+ */
+export const getAccount = async (addr: string) => {
+  const balance = await api.getBalance(addr);
+  const nonce = await api.getNonce(addr);
+  const blockHeight = await api.getBlockchainBlockHeight();
```

Comment:
It is used in confirmartion of operation logic. 
In ledger an operation is considered confirmed only if account blockHeight is greater than operation blockHeight. So that I need to constantly set the account blockHeight at current blockchain blockHeight.